core/modules/user/user.module

```diff
@@ -3186,9 +3189,10 @@ function user_register_validate($form, &$form_state) {
  * @see user_register_form()
  */
 function user_register_submit($form, &$form_state) {
+  $site_config = config('system.core');
```

Comment:
Hm... it seems, I repeat myself... :laughing: 

As 'user_email_verification' is the only thing needed in this function, you could:
```
$user_email_verification = config_get('system.core', 'user_email_verification');
```
And use that variable later on.

Reply:
Done!